### DISCUSSION TITLE
Ntrig driver on brunch OS

### DISCUSSION THREAD

**Discussion by sskye123** [sebanc/brunch#1717] [General]
Hi, I am using surface go 3 and it seems there is some latency using surface pen. Does brunch os use the Ntrig Linux driver in the kernel as I believe the surface pen for go 3 is based on Ntrig technology? Thank you

**Comment by zampemo** [comment]
Hi, how is it performing? Is everything working? Which base did you choose to install? I've the Go 3 with the Pentium Gold, thx!
